releases.md

```diff
@@ -41,11 +41,17 @@ Further documentation available:
 
 ## Release
 
-### v0.61
-- **Latest Release**: [v0.61.0][v0.61-0] (2024-06-25) ([docs][v0.61-0-docs], [examples][v0.61-0-examples])
-- **Initial Release**: [v0.61.0][v0.61-0] (2024-06-25)
+### v0.63
+- **Latest Release**: [v0.63.0][v0.63-0] (2024-08-30) ([docs][v0.63-0-docs], [examples][v0.63-0-examples])
+- **Initial Release**: [v0.63.0][v0.63-0] (2024-08-30)
+- **End of Life**: 2024-09-30
+- **Patch Releases**: [v0.63.0][v0.63-0]
+
+### v0.62 (LTS)
+- **Latest Release**: [v0.62.2][v0.62-2] (2024-08-23) ([docs][v0.62-2-docs], [examples][v0.62-2-examples])
+- **Initial Release**: [v0.62.0][v0.62-0] (2024-07-23)
 - **End of Life**: 2024-07-25
```

Comment:
EOL for v0.62 should be 1 year after release as it's an LTS release, i.e. 2025-07-23

Reply:
Fixed!